source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -425,11 +434,13 @@ namespace AdaptiveCards { namespace Uwp
 
     _Use_decl_annotations_
     template<typename T>
-    void XamlBuilder::SetImageOnUIElement(_In_ ABI::Windows::Foundation::IUriRuntimeClass* imageUri, T* uiElement, IAdaptiveRenderContext* renderContext)
+    void XamlBuilder::SetImageOnUIElement(
+        _In_ ABI::Windows::Foundation::IUriRuntimeClass* imageUri,
+        T* uiElement,
+        IAdaptiveCardResourceResolvers* resolvers)
     {
         // Get the resource resolvers
-        ComPtr<IAdaptiveCardResourceResolvers> resolvers;
-        THROW_IF_FAILED(renderContext->get_ResourceResolvers(&resolvers));
+        ComPtr<IAdaptiveCardResourceResolvers> localResolvers(resolvers);
```

Comment:
> ComPtr<IAdaptiveCardResourceResolvers> localResolvers(resolvers); [](start = 7, length = 66)

Looks like this isn't used?

Reply:
Yep good catch. I added it at some point as I needed to do a .As, but I removed it at later.

---
In reply to: [149127556](https://github.com/Microsoft/AdaptiveCards/pull/880#discussion_r149127556) [](ancestors = 149127556)